app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventFragment.java

```diff
@@ -77,6 +91,30 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                 getPresenter().createEvent();
         });
 
+        binding.form.buttonPlacePicker.setOnClickListener(view -> {
+            int errorCode = GoogleApiAvailability.getInstance().isGooglePlayServicesAvailable(getContext());
+            if (errorCode == ConnectionResult.SUCCESS) {
+                //SUCCESS
+                PlacePicker.IntentBuilder builder = new PlacePicker.IntentBuilder();
+                try {
+                    startActivityForResult(builder.build(getActivity()), PLACE_PICKER_REQUEST);
+                } catch (GooglePlayServicesRepairableException e) {
+                    Log.e(LOG_TAG, "GooglePlayServicesRepairable", e);
+                } catch (GooglePlayServicesNotAvailableException e) {
+                    Log.e(LOG_TAG, "GooglePlayServices NotAvailable => Updating or Unauthentic");
+                }
+            } else if (GoogleApiAvailability.getInstance().isUserResolvableError(errorCode)) {
+                //SERVICE_MISSING, SERVICE_VERSION_UPDATE_REQUIRED, SERVICE_DISABLED
+                GoogleApiAvailability.getInstance().getErrorDialog(
+                    getActivity(), errorCode, PLACE_PICKER_REQUEST, (dialog) -> {
+                        showLocationLayouts();
+                    });
+            } else {
+                //SERVICE_UPDATING, SERVICE_INVALID - can't use place picker - must enter manually
+                showLocationLayouts();
+            }
+        });
```

Comment:
Instance of GoogleApiAvailability should be stored in a global variable

Reply:
I have made the specified changes. Please review.